stats/tests/test_comprehensiveness.py

```diff
@@ -1428,9 +1428,157 @@ def test_comprehensivness_denominator_recipient_language_false(major_version):
     activity_stats.today = datetime.date(9990, 6, 1)
     activity_stats.element = etree.fromstring('''
         <iati-activity>
-            <activity-status code="2"/> 
+            <activity-status code="2"/>
             <recipient-country code="AF" percentage="50" />
             <recipient-country code="AG" percentage="50" />
         </iati-activity>
     ''')
     assert activity_stats.comprehensiveness_denominators()['recipient_language'] == 0
+
+
+@pytest.mark.parametrize('major_version', ['1', '2'])
```

Comment:
`transaction`s don't exist at V1. As such, it should be tested that this acts differently across major versions as appropraite.